qos/jsonrpc/params.go

```diff
@@ -0,0 +1,41 @@
+package jsonrpc
+
+import (
+	"encoding/json"
+	"errors"
+)
+
+// Params stores the data contained in the `params` field of a JSONRPC request.
+// As of PR #170, it supports:
```

Comment:
Should it also mention supporting array containing strings/booleans?

Reply:
Thanks, updated.